routers/home.go

```diff
@@ -380,6 +381,28 @@ func ExploreCode(ctx *context.Context) {
 	ctx.HTML(200, tplExploreCode)
 }
 
+// ExploreAvatars redirect browser to user avatar of requested size
+func ExploreAvatars(ctx *context.Context) {
+	userName := ctx.Params(":username")
+	size, err := strconv.Atoi(ctx.Params(":size"))
+	if err != nil {
+		ctx.ServerError("Invalid avatar size", err)
+		return
+	}
+
+	log.Debug("Asked avatar for user %v and size %v", userName, size)
+
+	user, err := models.GetUserByName(userName)
+	if err != nil {
+		// if models.IsErrUserNotExist(err)
+		ctx.ServerError("Requested avatar for invalid user", err)
+		return
+	}
+
+	url := user.RealSizedAvatarLink(size)
```

Comment:
No need for variable if it is used only once, just:
```suggestion
	ctx.Redirect(user.RealSizedAvatarLink(size))
```